spec/fixtures/pdf_fill/21P-527EZ/kitchen_sink.json

```diff
@@ -100,7 +100,7 @@
   ],
   "spouseDateOfBirth": "1960-01-01",
   "spouseSocialSecurityNumber": "333224444",
-  "spouseIsVeteran": true,
+  "spouseIsVeteran": false,
```

Comment:
Changed this to false to give the currently expected PDF output (I'm not able to update the PDFs at this time). We can flip this and update the PDF when less time-constrained if desired